content/flash-firmware-from-usb.md

```diff
@@ -46,6 +46,18 @@ The highlighted files pictured below (the hash will change based on file and mod
 
 ![files-unzipped](/images/open-firmware-smmstore/files-unzipped.png)
 
+## Extracting an img to the USB drive
+
+Firmware is typically packaged in the form of a zip archive, but support will ocasionally provide an .img image. If the firmware image provided is a .img file, it will need to be flashed using the Disks application. We do this by "restoring" the image to the drive.
```

Comment:
Think it's worth mentioning Popsicle? It's quite a bit more user-friendly than using Disks.

Reply:
I think we should include Popsicle. Yes.

Reply:
To be honest, I didn't consider that. I pulled most of the text from the live disk article, which we'll also be re-working. I agree with this change. I'll work on this.